tests/e2e/test_io_performance.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module to perform IOs with several weights
+"""
+import pytest
+import logging
+from ocs_ci.utility.spreadsheet.spreadsheet_api import GoogleSpreadSheetAPI
+
+from ocs_ci.framework.testlib import ManageTest, tier1, google_api_required
+from tests.fixtures import (
+    create_rbd_storageclass, create_rbd_pod, create_pvc, create_ceph_block_pool,
+    create_rbd_secret, create_project
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+    create_project.__name__,
+    create_pvc.__name__,
+    create_rbd_pod.__name__,
+)
+@google_api_required
+@tier1
+class TestIOPerformance(ManageTest):
+    """
+    Test IO performance
+    """
+
+    @pytest.mark.parametrize(
+        argnames=[
+            "size", "io_direction", "jobs", "runtime", "depth",
+            "cell_to_update"
+        ],
+        argvalues=[
+            pytest.param(
+                *['1GB', 'rw', 1, 120, 4, (5, 2)],
+                #marks=pytest.mark.polarion_id("OCS-555")
+            ),
+            pytest.param(
+                *['1GB', 'rw', 6, 120, 16, (13, 2)],
+                #marks=pytest.mark.polarion_id("OCS-558")
+            ),
+            pytest.param(
+                *['1GB', 'rw', 12, 120, 32, (21, 2)],
+                #marks=pytest.mark.polarion_id("OCS-559")
+            ),
+        ]
+    )
+    def test_run_io(
```

Comment:
giving this new test, do we still need tests.manage.test_pvc_deletion_during_io.TestDeletePVCWhileRunningIO#test_run_io?

Reply:
No, I will remove it